pkg/cri/config/config_windows.go

```diff
@@ -54,7 +54,7 @@ func DefaultConfig() PluginConfig {
 			TLSKeyFile:  "",
 			TLSCertFile: "",
 		},
-		SandboxImage:            "mcr.microsoft.com/oss/kubernetes/pause:1.4.1",
+		SandboxImage:            "k8s.gcr.io/pause:3.4.1",
```

Comment:
Just curious, shouldn't this be pointing to the `microsoft` registry?

Reply:
nope!

Reply:
it's a multi-arch image
```
docker run --rm mplatform/mquery k8s.gcr.io/pause:3.4.1
Image: k8s.gcr.io/pause:3.4.1
 * Manifest List: Yes
 * Supported platforms:
   - linux/amd64
   - linux/arm/v7
   - linux/arm64
   - linux/ppc64le
   - linux/s390x
   - windows/amd64
   - windows/amd64
   - windows/amd64
   - windows/amd64
   - windows/amd64

```

Reply:
ah thanks.

Reply:
Just to complete this comment, a newer pre-release version of mplatform/mquery correctly reports the Windows OS variants:
```
> docker run --rm mplatform/mquery:v0.3 k8s.gcr.io/pause:3.4.1
Image: k8s.gcr.io/pause:3.4.1 (digest: sha256:6c3835cab3980f11b83277305d0d736051c32b17606f5ec59f1dda67c9ba3810)
 * Manifest List: Yes (Image type: application/vnd.docker.distribution.manifest.list.v2+json)
 * Supported platforms:
   - linux/amd64
   - linux/arm/v7
   - linux/arm64
   - linux/ppc64le
   - linux/s390x
   - windows/amd64:10.0.17763.1697
   - windows/amd64:10.0.18362.1256
   - windows/amd64:10.0.18363.1256
   - windows/amd64:10.0.19041.746
   - windows/amd64:10.0.19042.746
```

i.e. RS5 (Server LTSC 2019), 19H1, 19H2, 20H1, 20H2 are the supported Windows versions, since Hyper-V virtualisation support has been dropped by k8s.